specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -5318,543 +6439,463 @@
+          "description": "Details of the transfer.",
           "x-ms-client-flatten": true,
-          "$ref": "#/definitions/AvailableBalanceProperties",
-          "title": "Available balance properties"
+          "$ref": "#/definitions/TransferProperties"
         }
       }
     },
-    "AvailableBalanceProperties": {
-      "description": "The properties of available balance.",
+    "TransferProperties": {
+      "description": "Transfer details",
+      "type": "object",
       "properties": {
-        "amount": {
-          "description": "Balance amount.",
+        "creationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time at which the transfer request was created.",
+          "readOnly": true
+        },
+        "expirationTime": {
+          "type": "string",
+          "format": "date-time",
           "readOnly": true,
-          "$ref": "#/definitions/Amount"
+          "description": "The time at which the transfer request expires."
+        },
+        "invoiceSectionId": {
+          "type": "string",
+          "description": "The ID of the invoice section to which the product is billed after the transfer request is completed.",
+          "readOnly": true
+        },
+        "billingAccountId": {
+          "type": "string",
+          "description": "The ID of the billing account to which the product is billed after the transfer request is completed.",
+          "readOnly": true
+        },
+        "resellerId": {
+          "type": "string",
+          "description": "Optional MPN ID of the reseller for transfer requests that are sent from a Microsoft Partner Agreement billing account.",
+          "readOnly": true
+        },
+        "resellerName": {
+          "type": "string",
+          "description": "Optional name of the reseller for transfer requests that are sent from Microsoft Partner Agreement billing account.",
+          "readOnly": true
+        },
+        "initiatorCustomerType": {
+          "type": "string",
+          "description": "The type of customer who sent the transfer request.",
+          "readOnly": true
+        },
+        "billingProfileId": {
+          "type": "string",
+          "description": "The ID of the billing profile to which the product will be billed after the transfer.",
+          "readOnly": true
+        },
+        "transferStatus": {
+          "type": "string",
+          "$ref": "#/definitions/TransferStatus",
+          "description": "Overall transfer status.",
+          "readOnly": true
+        },
+        "recipientEmailId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The email ID of the user to whom the transfer request was sent."
+        },
+        "initiatorEmailId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The email ID of the user who sent the transfer request."
+        },
+        "canceledBy": {
```

Comment:
isn't cancelling done by deleting the transfer?  How would someone see this after its deleted?

Reply:
Cancelled transfer are shown with Status as 'Canceled' on azure portal.

Reply:
This won't work. After a successful DELETE call on a resource, the subsequent GET should return 404. See https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#delete-resource.

Reply:
As per our use case and current implementation (for preview versions) a transfer reqeust is never deleted. Is it possible to have more discussion on it in scheduled meeting?

Reply:
A DELETE request that doesn't actually delete the resource is not allowed. The only 2 valid options here are:
1. Stop returning the resource on GETs after a successful DELETE request.
2. Have a POST /cancel operation that sets canceled state on the resource.

If you can't do option 1, you will need to do option 2.